connector-webhook/src/main/java/com/redhat/cloud/notifications/connector/webhook/WebhookExceptionProcessor.java

```diff
@@ -23,12 +26,16 @@ public class WebhookExceptionProcessor extends ExceptionProcessor {
     @Override
     protected void process(Throwable t, Exchange exchange) {
         if (t instanceof HttpOperationFailedException e) {
+            log(DEBUG, e, exchange);
+            exchange.setProperty(HTTP_STATUS_CODE, e.getStatusCode());
             if (http400ErrorRange.isValidValue(e.getStatusCode())) {
-                // TODO Disable the integration using the 'integration-disabled' event type.
-                log(DEBUG, e, exchange);
+                exchange.setProperty(DISABLE_ENDPOINT_CLIENT_ERRORS, Boolean.TRUE);
             } else {
-                log(ERROR, e, exchange);
+                exchange.setProperty(INCREMENT_ENDPOINT_SERVER_ERRORS, Boolean.TRUE);
             }
+        } else if (t instanceof SocketTimeoutException
+            || t instanceof UnknownHostException) {
```

Comment:
We could include all `IOException` here:
```suggestion
        } else if (t instanceof IOException) {
```